### PR TITLE
Big update

### PR DESCRIPTION
**Pull request by Mivgander**
Hey!
I really like your site so I decited to upgrade it as good as I can.
Basically I just cleaned your code, changed type of showing recipies and added features.
I hope you accept my request.

P. S.
If you accept this request you will have to change mod in isaac, because instead of 'data' it is now using 'seed' for seed and 'coop' for coop e.g.
if coop is on: `seed=rb9bgjjj&coop=on`
if coop is disable there is no coop variable
I recommend checking everything to make sure everything is ok

### CODE REVIEW

File: static/app.js [hunk 1]
```diff
@@ -1,573 +1,186 @@
...
     var ret = {};
     for(var key in json){
       ret[json[key]] = key;
     }
     return ret;
-  }
+}
 
-let first_time = true;
+// Idk why this function is here but why not
```
Comment (heyngra) [inline static/app.js]: it was used before, left it so if I mess something up I can just yoink the code

File: templates/index.html [hunk 2]
```diff
@@ -1,37 +1,50 @@
...
+        A lot of code is taken from <a href="https://github.com/frto027/IsaacDisassemblyRate" target="_blank">this repository</a>
+    </div>
     <div class="page-main">
         <h1>Tainted Cain Recipes</h1>
-        <input id="search" class="searchbar"></input>
+        <p class="note">
+            <b>NOTE!</b><br>
+            The number of recipes is limited to 10 per item for efficiency reasons.<br>
+            Remember that each item has a lot more recipes. If you want to find another <span onclick="refreshPage()" class="clickable">refresh page</span>
+        </p>
```
Comment (heyngra) [inline templates/index.html]: this is probably overwritten by some other party of code, I don't even see it after page reload

**Review by heyngra — COMMENTED** [review]
this is not working

File: templates/index.html (hunk 2 above)
Comment (Mivgander) [reply templates/index.html]: I added this in one commit and deleted in another

**Comment by Mivgander** [comment]
For testing final page use files in my branch development. Don't worry about old commits

**Comment by heyngra** [comment]
when you click 2 times resume (for example cuz of double click) then pause don't work

**Comment by heyngra** [comment]
https://tcain.heyn.live/?seed=JKD9Z0C9 crafts is not defined ???

**Comment by heyngra** [comment]
image

this happens when you actually CAN fetch, for example by adding `chrome.exe --disable-web-security --user-data-dir="/temp/chrome_dev"` or launching it from tcain.heyn.live)

**Comment by heyngra** [comment]
polska gurom

**Comment by heyngra** [comment]
polska gurom

**Pull request merged**